packages/expo-camera/android/src/main/java/expo/modules/camera/tasks/ResolveTakenPicture.kt

```diff
@@ -271,3 +266,17 @@ class ResolveTakenPicture(
     else -> 0
   }
 }
+
+@Throws(Exception::class)
+fun writeStreamToFile(directory: File, inputStream: ByteArrayOutputStream): String? {
+  try {
+    val outputPath = FileSystemUtils.generateOutputPath(directory, DIRECTORY_NAME, EXTENSION)
+    FileOutputStream(outputPath).use { outputStream ->
+      inputStream.writeTo(outputStream)
+    }
+    return outputPath
+  } catch (e: IOException) {
+    e.printStackTrace()
```

Comment:
We should throw something here because we're currently suppressing the `IOException`.